rmw_connextdds_common/src/common/rmw_impl.cpp

```diff
@@ -442,8 +441,8 @@ rmw_connextdds_get_readerwriter_qos(
         liveliness->kind = DDS_MANUAL_BY_TOPIC_LIVELINESS_QOS;
         break;
       }
-    // case RMW_QOS_POLICY_LIVELINESS_UNKNOWN:
-    default:
```

Comment:
Is there any possibility that a user can pass an invalid value for these enums (like some random value that doesn't match any name in the enum)? This question goes for other switchs without a default clause.